files/zh-cn/web/http/headers/access-control-expose-headers/index.md

```diff
@@ -26,26 +26,42 @@ slug: Web/HTTP/Headers/Access-Control-Expose-Headers
 ## 语法
 
 ```plain
-Access-Control-Expose-Headers: <header-name>, <header-name>, ...
+Access-Control-Expose-Headers: [<header-name>[, <header-name>]*]
+Access-Control-Expose-Headers: *
 ```
 
 ## 指令
 
 - \<header-name>
   - : 包含 0 个或多个除 {{Glossary("Simple response header", "simple response headers")}}（简单响应首部）之外的[首部名称](/zh-CN/docs/Web/HTTP/Headers)列表，可以暴露给外部，供页面资源使用。
 
+- \*（通配符）
+  - : 若请求没有 [HTTP Cookie](/zh-CN/docs/Web/HTTP/Cookies)或认证信息，通配符 ”*“ 会暴露所有响应头部。对于有凭证信息的请求，会被简单地识别为头部名 ”*“，没有通配的语义。
+
 ## 示例
 
 想要暴露一个非简单响应首部，可以这样指定：
 
 ```plain
-Access-Control-Expose-Headers: Content-Length
+Access-Control-Expose-Headers: Content-Encoding
+```
+
+想要额外暴露自定义的首部，例如 `Kuma-Revision`，可以指定多个，用逗号隔开：
+
+```plain
+Access-Control-Expose-Headers: Content-Encoding, Kuma-Revision
 ```
 
-想要额外暴露自定义的首部，例如 `X-Kuma-Revision`，可以指定多个，用逗号隔开：
+有凭证信息的请求，可以指定一个通配符：
 
 ```plain
-Access-Control-Expose-Headers: Content-Length, X-Kuma-Revision
+Access-Control-Expose-Headers: *
+```
+
+但是，“*” 对 [Authorization 头](/zh-CN/docs/Web/HTTP/Headers/Authorization) 不生效，需要显式指定：
```

Comment:
See line: https://github.com/mdn/content/blob/2344afbffd22b7932b755d6c74b8242ededdfbd2/files/en-us/web/http/headers/access-control-expose-headers/index.md?plain=1#L65

Please reflect the translation as the english content

Reply:
```suggestion
但是，这并不会匹配 {{HTTPHeader("Authorization")}} 标头，所以如果你要暴露它，需要显式指定：
```